app/javascript/packs/dashboard.js

```diff
@@ -1,3 +1,5 @@
+import '@babel/polyfill'
```

Comment:
Add this to the babelrc, instead of every single module

Reply:
It has to be imported at the top of the application's entry point (which I believe in our case are the many packs), it's not the old polyfill [ -> read the docs](https://babeljs.io/docs/en/babel-polyfill/#usage-in-node-browserify-webpack).

Reply:
Then in webpack config:
> If useBuiltIns key is not specified or it is explicitly set with useBuiltIns: false in your .babelrc, add @babel/polyfill directly to the entry array in your webpack.config.js.